src/tests/syncEnrichers.test.ts

```diff
@@ -0,0 +1 @@
+// TODO: https://transcend.height.app/T-10530 - syncEnrichers test
```

Comment:
planning to add these test all in at the end once i get the whole thing working. will make sure its tested before publishing